recipes/pytorchrl/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pytorchrl" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pytorch/rl/archive/v{{ version }}.tar.gz
+  sha256: 8851a84316f2a1d61d23ec753c90b545bc6479890ec65c5278987ae7c8a2ceec
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]
+  missing_dso_whitelist:
+    - $RPATH/libtorch_python{{ SHLIB_EXT }}  # [not win]
```

Comment:
I'm not sure about this part. @isuruf or @jaimergp, sorry for the ping but it this OK to go?

Reply:
The same is used in a couple of packages on `conda-forge`:
* https://github.com/conda-forge/sinabs-exodus-feedstock
* https://github.com/conda-forge/theseus-ai-feedstock
* https://github.com/conda-forge/pytorch3d-feedstock

Reply:
Waiting for someone with more knowledge on this to chime in.